docs/docs/APIdocs/Layers/Recurrent-Layers.md

```diff
@@ -9,7 +9,15 @@ val module = Recurrent()
 module = Recurrent()
 ```
 
-Recurrent module is a container of rnn cells. Different types of rnn cells can be added using add() function.
+Recurrent module is a container of rnn cells. Different types of rnn cells can be added using add() function.  
+
+Recurrent support return final state and cell status of its rnn cells by using getFinalStateAndCellStatus. output of getFinalStateAndCellStatus
```

Comment:
I have added getFinalStateAndCellStatus in recurrent, which support return. Code has been merged, this pr adds some explaination